.github/workflows/release-android.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build release
```

Comment:
Could this name specify that it's the Android release?

Reply:
Yes, good point. Done.